doc/port.1.txt

```diff
@@ -129,7 +129,7 @@ The port command recognizes several global flags and options.
     Quiet mode, suppress informational messages to a minimum, implies -N
 
 -N::
-    Non-interactive mode, interactive questions are not asked 
+    Non-question mode, interactive questions are not asked (avoid typing yes)
```

Comment:
@jmroot Thank you for resolving the documentation conflict between shell mode and interactive mode.  However, can you also include the change from the end of this line `(avoid typing yes)`?  I searched `man port` for `yes` when looking for this option, and was unable to find it.